front/app/components/Form/Components/Controls/MultiSelectControl.tsx

```diff
@@ -25,6 +25,7 @@ import { getInstructionMessage } from './utils';
 
 const StyledMultipleSelect = styled(MultipleSelect)`
   flex-grow: 1;
+  z-index: 1001;
 `;
```

Comment:
Normally I'd just merge this change, but wanted to double-check in case you're aware of any potential situation where this wouldn't be desirable :) It seems to fix the issue, and I checked on mobile and the registration form as well.